tests/__tests__/api/article/searchTitle.js

```diff
@@ -0,0 +1,95 @@
+// searchArticles.test.js
+import request from "supertest";
+import app from "../../../../app/app.js";
+import { validSearchQuery, validSearchQuery2, validSearchQuery3, InvalidSearchQuery, InvalidSearchQuery2 } from "../../../testData/articleTestData.js";
+import Connection from "../../../../app/db/connection.js";
+import { log } from "../../../testConfig.js";
+
+const showLog = __filename
+  .replace(".js", "")
+  .split(/[/\\]/)
+  .splice(__filename.split(/[/\\]/).lastIndexOf("__tests__") + 1)
+  .reduce((acc, key) => acc && acc[key], log);
+
+beforeAll(async () => {
+  await Connection.open(true);
+});
+
+afterAll(async () => {
+  await Connection.close(true);
+});
+
+/*
+These tests pass only when using the cdi tool to connect to the remote atlas database
+This is due to the fact that fuzzy search is a only an atlas feature
+As these tests will not pass in local environments, they are marked here as skipped by default
+*/
+describe("Search Articles", () => {
+  test.skip("Valid fuzzy search query of title", async () => {
+    const response = await request(app).get(`/articles/search/title`).query({ search: validSearchQuery });
+
+    showLog && console.log(response.body);
+    expect(response.status).toBe(200);
+    expect(response.body.length).toBeGreaterThan(0);
```

Comment:
Expecting the length of the body to be greater than zero is okay, but semantically its hard to understand what the response is. Similar to the comment above about output validation, this doesn't really tell us if the fuzzy search returned the correct article/records, but that it returned something, in general, with length greater than zero. 

It doesn't have to exactly match records -- I can see why you wrote the tests like this, so that any edits to the articles or content wouldn't affect it. 

It's my philosophy that the reason why we have tests is to make sure other things aren't affected by things that we do, and if test data is edited, then we should expect different things, and that's okay. But thats an opinion.